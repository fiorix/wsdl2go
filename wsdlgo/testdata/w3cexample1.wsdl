<definitions>

<message name="getTermRequest">
  <part name="term" type="xs:string"/>
</message>

<message name="getTermResponse">
  <part name="value" type="xs:string"/>
</message>

<portType name="glossaryTerms">
  <operation name="getTerm">
    <input message="getTermRequest"/>
    <output message="getTermResponse"/>
  </operation>
</portType>

</definitions>
