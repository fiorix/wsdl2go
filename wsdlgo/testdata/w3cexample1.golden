package internal

import (
	"context"
	"errors"
)

// GetNothing was auto-generated from WSDL.
func GetNothing(ctx context.Context, something string) error {
	return errors.New("not implemented")
}

// GetTerm was auto-generated from WSDL.
func GetTerm(ctx context.Context, term string) (string, error) {
	return "", errors.New("not implemented")
}

// GetVoid was auto-generated from WSDL.
func GetVoid(ctx context.Context, something string) error {
	return errors.New("not implemented")
}
